packages/file_selector/file_selector_platform_interface/lib/src/platform_interface/file_selector_interface.dart

```diff
@@ -74,4 +74,13 @@ abstract class FileSelectorPlatform extends PlatformInterface {
   }) {
     throw UnimplementedError('getDirectoryPath() has not been implemented.');
   }
+
+  /// Open file dialog for loading directories and return multiple directories paths
+  /// Returns `null` if user cancels the operation.
+  Future<List<String>?> getDirectoriesPaths({
```

Comment:
This should not be nullable; cancelling for multi-select is indicated with an empty List. Having two ways of expressing the same thing (null and empty) makes APIs error-prone to use.